.changeset/eighty-dolphins-admire.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think the changeset can be empty since it was already omitted from the dist :)

Reply:
Good point 🤔